pandas/tests/window/conftest.py

```diff
@@ -308,3 +308,34 @@ def which(request):
 def halflife_with_times(request):
     """Halflife argument for EWM when times is specified."""
     return request.param
+
+
+@pytest.fixture(
```

Comment:
it might be worth putting this in pandas/conftest.py (maybe), but can try in a followon